.github/dependabot.yaml

```diff
@@ -1,7 +1,7 @@
 version: 2
 updates:
-- package-ecosystem: npm
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
\ No newline at end of file
+  - package-ecosystem: npm
```

Comment:
You don't have any `npm` package. https://gist.github.com/magnetikonline/6f215db058e327905bce66c37f92426c